R/zchunk_LA121.oil.R

```diff
@@ -35,70 +35,145 @@ module_energy_LA121.oil_DISABLED <- function(command, ...) {
     # Load required inputs
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     calibrated_techs <- get_data(all_data, "energy/calibrated_techs")
-    IEA_product_rsrc <- get_data(all_data, "energy/IEA_product_rsrc")
+    IEA_product_rsrc <- get_data(all_data, "energy/mappings/IEA_product_rsrc")
     A21.unoil_demandshares <- get_data(all_data, "energy/A21.unoil_demandshares")
     A21.globaltech_coef <- get_data(all_data, "energy/A21.globaltech_coef")
-    L100.IEA_en_bal_ctry_hist <- get_data(all_data, "temp-data-inject/L100.IEA_en_bal_ctry_hist")
-    L1011.en_bal_EJ_R_Si_Fi_Yh <- get_data(all_data, "L1011.en_bal_EJ_R_Si_Fi_Yh")
-    L111.Prod_EJ_R_F_Yh <- get_data(all_data, "L111.Prod_EJ_R_F_Yh")
+
+   L100.IEA_en_bal_ctry_hist <- get_data(all_data, "L100.IEA_en_bal_ctry_hist") %>%
+      gather(year, value, -iso, -FLOW, -PRODUCT) %>%
+      mutate(year = sub("X", "", year)) %>%
```

Comment:
?  Why is this being done? There should be no "xyears" coming in from the new data system.